compiler/qsc_eval/src/backend.rs

```diff
@@ -393,6 +396,26 @@ impl Backend for SparseSim {
             | "AccountForEstimatesInternal"
             | "BeginRepeatEstimatesInternal"
             | "EndRepeatEstimatesInternal" => Some(Ok(Value::unit())),
+            "ConfigurePauliNoise" => {
```

Comment:
IIRC we need to handle custom intrinsics in other `Backend` implementations as well, perhaps by no-opping them. To check, you can try generating a Circuit. How does it handle this custom call?

Reply:
Yes, I agree, these new noise intrinsics should be recognized and treated as no-op in circuit generation, partial evaluation, and resource estimation.

Reply:
I believe I updated them all.
Ideally, we shouldn't update other backends when we add a custom intrinsic to one. But we aren't ready for this yet with our current architecture. We discussed this with Mine.